packages/base-styles/_mixins.scss

```
@@ -367,6 +367,15 @@
 	}
 }
 
+@mixin link-reset {
+	&:focus {
+		color: var(--wp-admin-theme-color--rgb);
+		/* stylelint-disable-next-line declaration-property-value-disallowed-list */
+		box-shadow: 0 0 0 var(--wp-admin-border-width-focus) var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
```

```suggestion
		box-shadow: 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color, #007cba);
```

I think it is better not to include the `--wp-components-color-accent` variable since the `base-styles` package may be used even in projects that do not include the `component` package.